tests/integration_tests/data_steward/retraction/retract_utils_test.py

```diff
@@ -0,0 +1,62 @@
+# Python imports
+import os
+from unittest import TestCase
+
+# Third party imports
+import google.cloud.bigquery as gbq
+
+# Project imports
+import app_identity
+import retraction.retract_utils as ru
+
+
+class RetractUtilsTest(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+    def setUp(self):
+        self.project_id = app_identity.get_application_id()
+        self.dataset_id = os.environ.get('UNIONED_DATASET_ID')
+        self.client = gbq.Client()
+
+    def test_is_labeled_deid(self):
```

Comment:
I'll keep the integration test here since it helped me realize `True` was not a valid label value. BQ does not allow caps for labels. I'll move the other test to unit tests.